mathics/builtin/box/image.py

```diff
@@ -13,18 +32,80 @@ class ImageBox(BoxExpression):
     an Image object.
     """
 
+    def boxes_to_b64text(self, elements=None, **options):
+        contents, size = self.boxes_to_png(elements, **options)
+        encoded = base64.b64encode(contents)
+        encoded = b"data:image/png;base64," + encoded
+        return encoded, size
+
+    def boxes_to_png(self, elements=None, **options) -> Tuple:
```

Comment:
The return type is more than just a generic tuple it is something like `Tuple[<type of contents>, Tuple[int, int]]`

Please add and fill in "type of contents".